src/aatoolbox/datasources/ecmwf_api/ecmwf_api_download_seasonal_monthly.py

```diff
@@ -109,11 +112,16 @@ def download(
         # TODO: it would probably be safer to also include the boundary coords
         # in the filename.. Just that the filename then gets massive
         output_filename = (
-            f"{iso3}_{SEAS_DIR}_{PRATE_DIR}_{date_forec.strftime('%Y-%m')}.nc"
+            f"{iso3}_{SEAS_DIR}_{PRATE_DIR}_{date_forec.strftime('%Y-%m')}"
         )
+        if area is not None:
+            output_filename += "_{area.get_filename_repr()}"
```

Comment:
@caldwellst just tagging you here because I  know you were adding coordinates to the ARC filenames, so that you know that that is now generalized in the toolbox so should be easy to use when refactoring ARC to toolbox